exec/java-exec/src/main/java/org/apache/drill/exec/ssl/SSLConfigServer.java

```diff
@@ -226,11 +230,26 @@ private String getHadoopConfigParam(String name) {
     return value;
   }
 
-  private String resolveHadoopPropertyName(String nameTemplate, Mode mode) {
-    return MessageFormat.format(nameTemplate, mode.toString().toLowerCase());
-  }
+  private String getPasswordConfigParam(String name, String hadoopName) {
```

Comment:
Same as `getPasswordStringProperty` from `SSLConfigClient`. Can it be refactored?

Reply:
Extracted common code, please take a look.

Reply:
Looks good :+1: